python/tvm/relay/backend/interpreter.py

```diff
@@ -23,27 +23,13 @@
 from . import _backend
 from .. import _make, analysis, transform
 from .. import module
-from ... import register_func, nd
+from ... import nd
 from ..base import NodeBase, register_relay_node
 from ..expr import Tuple, RefCreate, Call, Constant, GlobalVar, Function, const
 from ..scope_builder import ScopeBuilder
-from . import _vm
-
-class Value(NodeBase):
-    """Base class of all values.
-    """
-    @staticmethod
-    @register_func("relay.from_scalar")
-    def from_scalar(value, dtype=None):
-        """Convert a Python scalar to a Relay scalar."""
-        return TensorValue(const(value, dtype).data)
-
-    def to_vm(self):
-        return _vm._ValueToVM(self)
-
 
 @register_relay_node
-class TupleValue(Value):
+class TupleValue(NodeBase):
```

Comment:
Shouldn't this be Object? or the equivalent Object base class, seems wrong to inherit from node. cc @tqchen

Reply:
I noticed this as well. It is from here:

https://github.com/apache/incubator-tvm/blob/bc0274d307226408c69226cf922dd916d773e265/python/tvm/_ffi/node.py#L33

and used as the base by `Value` before so I just kept it.

Reply:
There is no Node anymore in the c++ side, see https://github.com/apache/incubator-tvm/pull/4603 and https://github.com/apache/incubator-tvm/issues/4647

So NodeBase was a legacy item that we might need to upgrade on the python side.

Reply:
Let us inherit from Object for now, and have a subsequent PR to cleanup the python side.

Also seems we might be able to use Array(or ADT, see discussion here https://discuss.tvm.ai/t/discuss-runtime-array-containers-array-adt-string/4582) for this after @wweic 's PR lands

Reply:
Yes, I tried it, but but looks to me that we have to merge nodebase to object first before I can use object directly as there are fields not implemented in object, like getattr, etc. Should I use nodebase first then we can refactor them together?

Reply:
I just added a commit to replace NodeBase with Object